src/ngMessageFormat/messageformat.js

```diff
@@ -0,0 +1,938 @@
+'use strict';
+
+var angular = window['angular'];
+
+var $$messageFormatMinErr = angular['$$minErr']('$$messageFormat');
+
+// NOTE: ADVANCED_OPTIMIZATIONS mode.
+//
+// This file is compiled with Closure compiler's ADVANCED_OPTIMIZATIONS flag! Be wary of using
+// constructs incompatible with that mode.
+
+/**
+ * @ngdoc service
+ * @name $$messageFormat
+ *
+ * @description
+ * Angular internal service to recognize MessageFormat extensions in interpolation expressions.
+ * For more information, see:
+ * https://docs.google.com/a/google.com/document/d/1pbtW2yvtmFBikfRrJd8VAsabiFkKezmYZ_PbgdjQOVU/edit
+ */
+
+var noop = angular['noop'],
+    isFunction = angular['isFunction'],
+    toJson = angular['toJson'];
```

Comment:
Yes.  I'm using the `ADVANCED_OPTIMIZATIONS` mode of the Closure compiler which will rewrite `angular.noop` into something like `angular.x`.  This module has limited interaction with outside code (and injection has existing array syntax to guard against minification) so there aren't too many places that do this kind of explicit lookup.